Games/Cosmic_Blast/README.md

```diff
@@ -30,7 +30,7 @@ Cosmic Blast is an exciting and fast-paced arcade game where the fate of Earth l
 ## **Screenshots 📸**
 
 <br>
-![Cosmic_Blast](https://github.com/kunjgit/GameZone/tree/main/Games/Cosmic_Blast/assets/Cosmic_Blast.png)
+![Cosmic_Blast](https://github.com/vansh-codes/GameZone/tree/main/Games/Cosmic_Blast/assets/Cosmic_Blast.png)
```

Comment:
need not to change the name , i think changing name will make the link unworkable

Reply:
Okay fixing

Reply:
@ishita-43 fixed, do check once again now!